crates/joystick/src/common.rs

```diff
@@ -1,75 +1,29 @@
-use evdev::KeyCode;
-use std::time::Duration;
-
 pub static TAP_DURATION: u64 = 50;
```

Comment:
Think this one is unused now.

Reply:
i have a feeling that github is doing some weird stuff - i deleted all of this, including the tap duration